### DISCUSSION TITLE
`mini.comment` keymappings?

### DISCUSSION THREAD

**Discussion by clauseggers** [LazyVim/LazyVim#2461] [Q&A] [answered]
I’ve been trying to find the keymaps for the `mini.comment` plugin, and in an earlier [answer](https://github.com/LazyVim/LazyVim/discussions/2415) it is stated that `<leader>gc` is used for `mini.comment`. This seems to have changed so that `<leader>gc` maps to Telescope [commits](https://www.lazyvim.org/keymaps#telescopenvim). There is no mention of `mini.comment` in the [Keymaps](https://www.lazyvim.org/keymaps) page.

What are the `mini.comment` keyboard mappings at the moment?

**Comment by dpetka2001** [comment, marked as answer]
Please read again the earlier answer that you linked #2415. Nowhere is stated anything about `<leader>gc`. Just press plain `gc` without the `<leader>` key. 

There's nothing in the keymaps in the docs, because Lazyvim doesn't set these keymaps. These are internal keymaps used by the plugin `mini.comment` itself. You can read about its default configuration in the repo's [docs](https://github.com/echasnovski/mini.comment?tab=readme-ov-file#default-config).

**Reply by clauseggers** [reply]
Doh! Thanks for replying.
